packages/connection/src/Manager.php

```diff
@@ -174,21 +174,6 @@ public function initialize_rest_api_registration_connector() {
 		new REST_Connector( $this );
 	}
 
-	/**
-	 * Initializes all needed hooks and request handlers. Handles API calls, upload
-	 * requests, authentication requests. Also XMLRPC options requests.
-	 * Fallback XMLRPC is also a bridge, but probably can be a class that inherits
-	 * this one. Among other things it should strip existing methods.
-	 *
-	 * @param Array $methods an array of API method names for the Connection to accept and
-	 *                       pass on to existing callables. It's possible to specify whether
-	 *                       each method should be available for unauthenticated calls or not.
-	 * @see Jetpack::__construct
-	 */
-	public function initialize( $methods ) {
-		$methods;
```

Comment:
weird

Reply:
I assume it was like that because it was both:
* Unimplemented yet.
* Satisfying the linter because `$methods` was "used".